latest/examples/multiscales_strict/multiscales_example_relative.json

```diff
@@ -0,0 +1,61 @@
+{
+    "multiscales": [
+        {
+            "version": "0.5-dev",
+            "name": "example",
+            "coordinateSystems" : [
+                {
+                    "name" : "exampleCoordinateSystem",
+                    "axes": [
+                        {"name": "t", "type": "time", "unit": "millisecond"},
+                        {"name": "c", "type": "channel"},
+                        {"name": "z", "type": "space", "unit": "micrometer"},
+                        {"name": "y", "type": "space", "unit": "micrometer"},
+                        {"name": "x", "type": "space", "unit": "micrometer"}
+                    ]
+                }
+            ],
+            "datasets": [
+                {
+                    "path": "0"
+                    // the transformation of other arrays are defined relative to this, the highest resolution, array
```

Comment:
The spec still says:

"Each dictionary in "datasets" MUST contain the field "coordinateTransformations", which contains a list of transformations that map the data coordinates to the physical coordinates (as specified by "axes") for this resolution level.
The transformations are defined according to [[#trafo-md]]. The transformation MUST only be of type `translation` or `scale`.
They MUST contain exactly one `scale` transformation that specifies the pixel size in physical units or time duration. If scaling information is not available or applicable for one of the axes, the value MUST express the scaling factor between the current resolution and the first resolution for the given axis, defaulting to 1.0 if there is no downsampling along the axis."

Reply:
@bogovicj, assuming I should go with the spec over the example here?

Out of interest, why does the pixel -> physical need to be replicated here?

Reply:
I think the spec just needs updating or clarifying at that point. Maybe now it's

"Each dictionary in "datasets" *except the first* MUST contain the field "coordinateTransformations"...